### PR TITLE
CMT4K_wiki: Update.

### PR DESCRIPTION
**Pull request by lzc731524299**
- update round of 8 stage match schedule and mappool.
- update round of 16 stage result.
- Group Stage result: integrate 8 tables into 1 for tidiness.
- other error fixes.

### CODE REVIEW

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 1]
```diff
@@ -44,29 +44,49 @@ Twitch: [aiyulu](https://www.twitch.tv/aiyulu).
 
 ## Match schedule
 
-### Round of 16
+### Round of 8
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: Better Quarterfinals?

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 2]
```diff
@@ -44,29 +44,49 @@ Twitch: [aiyulu](https://www.twitch.tv/aiyulu).
...
-| 20:00 | [Crz]ScSolAr | ![][CN] | ![][CN] | [Crz]Mix0130 | H |
-
+| 13:00 | Murasame | ![][CA] | ![][AU] | [Crz]Yukikaze- | T |
+| 15:00 | pikechu | ![][CN] | ![][CN] | [Crz]Mix0130 | U |
+| 20:00 | [Crz]Liusecy | ![][CN] | ![][MY] | Neokje | K |
+| 21:00 | - Choma - | ![][TW] | ![][CN] | [Crz]ScSolAr | L |
 
 ## Mappools
 
+### Round of 8
```
Comment (TPGPL) [inline wiki/Tournaments/CMT_4K/2017/en.md]: Quarterfinals

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 3]
```diff
@@ -44,29 +44,49 @@ Twitch: [aiyulu](https://www.twitch.tv/aiyulu).
...
+- [Demetori - Wind God Girl \[14\]](https://osu.ppy.sh/b/1306105)
+- [Gram - Sigmund \[Karasu's Leggendaria\]](https://osu.ppy.sh/b/1347575)
+- [Iced Blade feat. Rita - Sora no Kodou (short ver.) \[4K Liberation\]](https://osu.ppy.sh/b/942355)
+- [EYKHA & LOU & MERYL - Delta Decision \[Domination\]](https://osu.ppy.sh/b/1276313)
+- [t+pazolite - cheatreal \[TUTORIAL\]](https://osu.ppy.sh/b/873247)
+
+TB:
+- [Billain - Specialist \[Maestro\]](https://osu.ppy.sh/b/1216877)
+
 ### Round of 16
```
Comment (Repflez) [inline wiki/Tournaments/CMT_4K/2017/en.md]: The first instance of this was replaced to Quarterfinals. Should be done here too.

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 4]
```diff
@@ -140,68 +160,52 @@ TB:
 
 ## Match Results
 
-### Group Stage
+### Round of 16
```
Comment (Repflez) [inline wiki/Tournaments/CMT_4K/2017/en.md]: See above?

**Comment by lzc731524299** [comment]
The first instance was actually that "round of 16" ---update--> "round of 8"  ---rephrase--> "Quarterfinals"
@Repflez

**Comment by Repflez** [comment]
I see.

**Pull request merged**